acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/web3j/privacy/Ibft2PrivacyClusterAcceptanceTest.java

```diff
@@ -144,16 +144,25 @@ public void canInteractWithMultiplePrivacyGroups() {
                 bob.getEnclaveKey()));
 
     // alice gets receipt from charlie's interaction
-    final PrivateTransactionReceipt firstExpectedReceipt =
+    final PrivateTransactionReceipt aliceReceipt =
         alice.execute(privacyTransactions.getPrivateTransactionReceipt(firstTransactionHash));
 
+    final PrivateTransactionReceipt bobReceipt =
+        bob.execute(privacyTransactions.getPrivateTransactionReceipt(firstTransactionHash));
+    final PrivateTransactionReceipt charlieReceipt =
+        bob.execute(privacyTransactions.getPrivateTransactionReceipt(firstTransactionHash));
+
+    System.out.println(
```

Comment:
You left a `sys.out` here

Reply:
Also, are we using `bobReceipt` and `charlieReceipt` for anything?

Reply:
Used that for debugging :-)
Gone now!

Reply:
it's still there @pinges - do you have changes that aren't pushed?